common/image_u8_parallel.c

```diff
@@ -157,12 +158,8 @@ void image_u8_gaussian_blur_parallel(workerpool_t *wp, image_u8_t *im, double si
     for (int i = 0; i < ksz; i++)
         k[i] = dk[i]*255;
 
-    if (0) {
-        for (int i = 0; i < ksz; i++)
-            printf("%d %15f %5d\n", i, dk[i], k[i]);
-    }
     free(dk);
 
     image_u8_convolve_2D_parallel(wp, im, k, ksz);
     free(k);
-}
\ No newline at end of file
+}
```

Comment:
Add all these and other "fixup" changes into the commit that inroduced the changes.